dcs-packaging-tool/dcs-packaging-tool-gui/src/main/resources/org/dataconservancy/config/applicationContext.xml

```diff
@@ -135,7 +137,21 @@
     <property name="localCountryCode" value="${localCountryCode}"/>
   </bean>
 
+  <bean id="emailValidator" class="org.dataconservancy.packaging.gui.util.EmailValidator" />
```

Comment:
Do we have a place where we're going to document all these things that we're going to address later?